helm/ds/values.yaml

```diff
@@ -28,14 +29,14 @@ userstore:
 
 # If you wan to disable the cts backend, set to false.
 cts:
-  enabled: true
+  enabled: false
 
 image:
   repository: gcr.io/engineering-devops
   # For development we set this to Always to get the most current image.
   pullPolicy: Always
   #pullPolicy: IfNotPresent
-  tag:  6.5.0
+  tag:  alextest
```

Comment:
I assume this is temporary - you might want to add a TODO to change this when the image you want becomes available.